src/routes/renderRoutes.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../../sequelize/db");
+const routes = {};
+
+routes.postgres = async (req, res) => {
+  if (!req.session.username) return res.redirect("/");
+  const { Accounts } = db.getModels();
+  const userAccount = await Accounts.findOne({
+    where: {
+      username: req.session.username,
```

Comment:
Can we look up by id here: `req.session.id`

Reply:
Oh yes